sdk/servicefabric/servicefabric/README.md

```diff
@@ -19,8 +19,9 @@ npm install @azure/servicefabric
 
 ##### Install @azure/ms-rest-nodeauth
 
+- Please install minimum version of `"@azure/ms-rest-nodeauth": "^3.0.0"`.
 ```bash
-npm install @azure/ms-rest-nodeauth
+npm install @azure/ms-rest-nodeauth@"^3.0.0"
 ```
 
 ##### Sample code
```

Comment:
Can you run the below sample with the latest code and ensure it compiles and runs without errors?

Reply:
I have updated the samples